ArjunaJTA/jdbc/tests/classes/com/hp/mwtests/ts/jdbc/basic/ConnectionImpleTest.java

```diff
@@ -125,13 +124,14 @@ public void beforeCompletion() { }
             public void afterCompletion(int status) {
                 try {
                     connectionToTest.isClosed();
+                    connectionToTest.close();
```

Comment:
Why would somebody change it? It's a unit test designed to test one specific test and not the dependencies. That specific test purpose is to verify that when isClose is called on ConnectionImpl call is delegated to the underlying connection (which is also mocked). Before the test used to fail because of the issue you're fixing.